packages/config/config/devices/0x000c/hs-wd200.json

```diff
@@ -0,0 +1,125 @@
+// HomeSeer Technologies HS-WD200+
```

Comment:
These files were split by firmware version, but the files were identical. I recombined them.